server/db/migrations/mysql/000001_create_calls_channels.up.sql

```diff
@@ -1,7 +1,7 @@
 CREATE TABLE IF NOT EXISTS calls_channels (
     ChannelID varchar(26) PRIMARY KEY,
     Enabled BOOLEAN,
-    Props JSON
+    Props JSON NOT NULL
```

Comment:
@cpoile I fixed the migration, went with `NOT NULL` but didn't enforce a default at the schema level due to some [MySQL quirks](https://www.percona.com/blog/cant-we-assign-a-default-value-to-the-blob-text-geometry-and-json-data-types/) I'd rather not deal with. A default value is still enforced by store logic as during insert a `nil` map will be serialized to `null` so we should be good.

The only caveat is that this is not a backwards compatible change in terms of your local state but I wanted to avoid the less clean `COALESCE` (or adding a new migration) since we haven't released any of this yet so to fix your db you'll need to manually run something like:

```sql
UPDATE calls_channels SET Props = 'null' WHERE Props IS NULL;
```

Reply:
Of course 'null' is NOT NULL. 😝

Reply:
Yeah, `null` JSON literal :)